### DISCUSSION TITLE
What is Difference between atoi wallet (IOTA address) and CLI wallet address

### DISCUSSION THREAD

**Discussion by mahesh2392** [iotaledger/iota.c#462] [Q&A] [answered]
I am exploring IOTC C lib wallet and what I find wallet addresses are atoi1qpg2xkj66wwgn8p2ggnp7p582gj8g6p79us5hve2tsudzpsr2ap4s30j0g9 like this with index but at WASP node side I am using wasp-cli and there I am having wallet address like 1F4M6CtSJkgFFqcCvWzarupZJ9Rg46zQFiW1LFp4zjSpS what is difference between both address ?? IOTA C wallet is same as firefly or cli wallet ?? if not what is difference amongst them?? It is possible to send IOTAs from CLI wallet to IOTA.C wallet ??

**Comment by oopsmonk** [comment, marked as answer]
iota.c is a client lib for IOTA 1.5 and WASP is a node software for IOTA 2.0, both protocols and address schemes are different.
Currently, the IOTA 1.5 address scheme could be found in https://github.com/iotaledger/tips/blob/main/tips/TIP-0031/tip-0031.md
see https://wiki.iota.org/build/networks/iota-1.5 and https://wiki.iota.org/build/networks/iota-2.0
